ansible.cfg

```diff
@@ -0,0 +1,2 @@
+[defaults]
+hostfile       = hosts
```

Comment:
I don't think this belongs in this repo as it will never be used since this is pulled inside the role directory for most people.